### PR TITLE
Now it connects to Lobby Server

### PR DESCRIPTION
**Pull request by Deudly**
Now it connects to Lobby Server and some fix for connect to LobbyServer

### CODE REVIEW

**Comment by Deudly** [comment]
Now LobbyClient can access Lobbies

File: src/ui/services/networkService.ts [hunk 1]
```diff
@@ -60,14 +61,17 @@ export class NetworkService extends EventEmitter {
         
         this.currentUsername = username;
         return new Promise((resolve, reject) => {
-            this.currentLobbyConnection = io.connect(item.address + ":" + item.port, { reconnection: false });
+            this.currentLobbyConnection = io.connect(item.address + ":" + item.port, { reconnection: false, 'forceNew': true});
```
Comment (MythicManiac) [inline src/ui/services/networkService.ts]: Why is this change required?

Also, you should keep the dictionary consistent—you're missing a whitespace from the end. So either add one there or remove the one from beginning as well, personally I'd remove the one from the beginning.

So `{reconnection: false, 'forceNew': true}` instead of `{ reconnection: false, 'forceNew': true}`
